src/meta.jl

```diff
@@ -183,3 +183,7 @@ end
 function serve_parents(http::HTTP.Stream)
     return serve_json(http, config.storage_servers)
 end
+
+function serve_robots_txt(http::HTTP.Stream)
+    return write(http, "User-agent: * Disallow: /\n")
```

Comment:
Curious to ask, does linefeed matter in this case?

```diff
-     return write(http, "User-agent: * Disallow: /\n")
+     return write(http, "User-agent: * \nDisallow: /\n")
```

Reply:
No, I don't think it matters much.